lnwallet/commitment.go

```diff
@@ -137,6 +137,36 @@ func DeriveCommitmentKeys(commitPoint *btcec.PublicKey,
 	return keyRing
 }
 
+// ScriptInfo holds a redeem script and hash.
+type ScriptInfo struct {
+	// PkScript is the outputs' PkScript.
+	PkScript []byte
+
+	// WitnessScript is the full script required to properly redeem the
+	// output with PkSript. This field will only be populated if a PkScript
+	// is p2wsh or p2sh.
+	WitnessScript []byte
+}
+
+// CommitScriptToRemote creates the script that will pay to the non-owner of
+// the commitment transaction, adding a delay to the script based on the
```

Comment:
does it add the delay in this commit?

Reply:
No, that's why I wrote it as "adding a delay based on the channel type". Maybe that isn't very clear?